src/main/java/com/jcabi/log/VerboseProcess.java

```diff
@@ -249,8 +254,12 @@ private String stdout(final boolean check) {
             System.currentTimeMillis() - start
         );
         if (check && code != 0) {
+            String errorMsg = stdout;
+            if (stderr != null) {
+                errorMsg += "\n" + stderr;
```

Comment:
@diorcety why concatenate them? If the user wants the sterr output he can ask for it through a method.